Gemfile.lock

```diff
@@ -411,4 +413,4 @@ DEPENDENCIES
   zeroclipboard-rails (>= 0.1.2)
 
 BUNDLED WITH
-   1.15.1
+   1.15.4
```

Comment:
I don't think we're using bundler 1.15.4 yet (looks like we're on 1.15.1 on all versions of ruby according to puppet: https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_rbenv/manifests/all.pp) so we shouldn't bump this.  It's not a problem if you've locally upgraded your bundler, but it does mean you're not using the same version as the rest of the devs, or the ones on our deployed environments so you should take care when committing the `Gemfile.lock` to check that if BUNDLED_WITH is changed, it's in keeping with the version of bundler that is used on our servers.